Description: Do not build web REPL
Author: Sébastien Villemot <sebastien@debian.org>
Last-Update: 2013-02-06
Forwarded: not-needed
--- a/Makefile
+++ b/Makefile
@@ -53,7 +53,7 @@
 		  umfpack z openblas
 
 PREFIX ?= julia-$(JULIA_COMMIT)
-install: release webrepl
+install: release
 	@-$(MAKE) $(QUIET_MAKE)
 	@for subdir in "sbin" "bin" "etc" "libexec" $(JL_LIBDIR) $(JL_PRIVATE_LIBDIR) "share/julia" ; do \
 		mkdir -p $(PREFIX)/$$subdir ; \
